controllers/keystoneapi_controller.go

```diff
@@ -127,9 +146,7 @@ func (r *KeystoneAPIReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		instance.Status.Conditions.Init(&cl)
 
 		// Register overall status immediately to have an early feedback e.g. in the cli
-		if err := r.Status().Update(ctx, instance); err != nil {
-			return ctrl.Result{}, err
-		}
+		return ctrl.Result{}, r.Status().Update(ctx, instance)
```

Comment:
same, shouldn't we requeue here as well?

Reply:
I think update on ourselves will trigger a reconcile anyhow.
Also instead of doing an update here, I would just return and let the defer call do the update in a single place

Reply:
Going to try that